app/views/employees/index.html.erb

```diff
@@ -17,7 +17,7 @@
     <% @employees.each do |employee| %>
       <tr>
         <td><%= link_to employee.name, employee_path(employee) %></td>
-        <% if employee.status =="t"%>
+       <% if employee.status == true%>
```

Comment:
Check identation here and bellow. You are also missing a space before ` %>`

Reply:
Done :)